### PR TITLE
Doc all new v1.10.0 amendments

### PR DESCRIPTION
**Pull request by mDuo13**
Follow up on #1771 and #1759 with the rest of the v1.10.0 amendments.

- Add the other new amendments to the Known Amendments list
- Tweak the format of the info table per-amendment with slightly more consistent spacing
- Update CheckCashMakesTrustLine amendment status (this amendment went live without much fanfare back on Jan. 23)

### CODE REVIEW

File: content/concepts/consensus-network/amendments/known-amendments.md [hunk 1]
```diff
@@ -1040,6 +1100,25 @@ Allows pre-authorization of accounting relationships (zero-balance trust lines)
...
+[XRPFees]: #xrpfees
+
+| Amendment    | XRPFees |
+|:-------------|:--------|
+| Amendment ID | 93E516234E35E08CA689FA33A6D38E103881F8DCB53023F728C307AA89D515A7 |
+| Status       | In Development |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Simplifies transaction cost calculations to use XRP directly rather than calculating indirectly in "fee units" and translating the results to XRP. Updates all instances of "fee units" in the protocol and ledger data to be drops of XRP instead, including:
```
Comment (scottschurr) [inline content/concepts/consensus-network/amendments/known-amendments.md:1113]: I want @ximinez to look at this description.  The amendment was recently made backward compatible with the API, so there might not be any replaced fields, only additional fields.  I was not part of the code review, so I have no actual details.

File: content/concepts/consensus-network/amendments/known-amendments.md [hunk 2]
```diff
@@ -77,10 +81,10 @@ The following is a comprehensive list of all known [amendments](amendments.html)
 ## CheckCashMakesTrustLine
 [CheckCashMakesTrustLine]: #checkcashmakestrustline
 
-| Amendment | CheckCashMakesTrustLine |
-|:----------|:-----------|
+| Amendment    | CheckCashMakesTrustLine |
```
Comment (intelliot) [inline content/concepts/consensus-network/amendments/known-amendments.md:84]: It's fine this time, but ideally, whitespace changes would be in a separate PR - so as to reduce noise when reviewing the salient changes.

File: content/concepts/consensus-network/amendments/known-amendments.md [hunk 3]
```diff
@@ -704,13 +734,28 @@ This amendment removes two trust lines from an account to itself that were creat
...
+| Amendment    | fixUniversalNumber |
+|:-------------|:-------------------|
+| Amendment ID | 2E2FB9CF8A44EB80F4694D38AADAE9B8B7ADAFD2F092E10068E61C98C4F092B0 |
+| Status       | In Development |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Simplifies and unifies the code for decimal floating point math. In some cases, this provides slightly better accuracy than the previous code, resulting in calculations whose least significant digits are different than when calculated with the previous code. The different results may cause other edge case differences where precise calculations are used, such as ranking of Offers or processing of payments that use several different paths.
+
+Without this amendment, the code continues to use separate calculations for `STAmount` and `IOUAmount` objects, and [Automated Market Maker (XLS-30d)](https://github.com/XRPLF/XRPL-Standards/discussions/78) uses a third class for calculations.
```
Comment (intelliot) [inline content/concepts/consensus-network/amendments/known-amendments.md:749]: @HowardHinnant to confirm

**Review by intelliot — APPROVED** [review]

**Review by scottschurr — APPROVED** [review]
I only reviewed the new descriptions.  There was one description that I'd like @ximinez to weigh in on.  But, with that one exception, what I saw looks good to me.  :+1:

File: content/concepts/consensus-network/amendments/known-amendments.md (hunk 3 above)
Comment (HowardHinnant) [reply content/concepts/consensus-network/amendments/known-amendments.md:749]: Looks good.

**Review by amarantha-k — APPROVED** [review]
lgtm

File: content/concepts/consensus-network/amendments/known-amendments.md (hunk 1 above)
Comment (mDuo13) [reply content/concepts/consensus-network/amendments/known-amendments.md:1113]: Asked him on Slack and he says, "Yeah, that looks right."

Comment (ximinez) [reply content/concepts/consensus-network/amendments/known-amendments.md:1113]: With the fix, there are no RPC API changes. The ledger entry and transaction field changes described are correct.
